anyway/widgets/suburban_widgets/accident_count_by_car_type_widget.py

```diff
@@ -1,16 +1,21 @@
 import logging
 from collections import defaultdict
 from functools import lru_cache
+from math import floor
 from typing import Dict
+
 from flask_babel import _
+
+import anyway.widgets.widget_utils as widget_utils
 from anyway.backend_constants import BE_CONST
 from anyway.infographics_dictionaries import segment_dictionary
-from anyway.models import InvolvedMarkerView
+from anyway.models import VehicleMarkerView
+from anyway.request_params import RequestParams
 from anyway.vehicle_type import VehicleCategory
-from anyway.widgets.widget import register
 from anyway.widgets.suburban_widgets.sub_urban_widget import SubUrbanWidget
-from anyway.request_params import RequestParams
-import anyway.widgets.widget_utils as widget_utils
+from anyway.widgets.widget import register
+
+TOP_3 = 2

```

Comment:
I suggest to change the name of this constant to express its usage, rather than its content, like number-of-items or something